app/src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,9 @@
         <activity android:name=".view.activities.RequestDetailActivity" />
         <activity android:name=".view.activities.MemberProfileActivity" />
         <activity android:name=".view.activities.SendRequestActivity" />
-        <activity android:name=".view.activities.SettingsActivity"/>
+        <activity android:name=".view.activities.SettingsActivity" />
+        <activity
+            android:name=".view.activities.AboutActivity" />
```

Comment:
I saw that you are providing Up navigation in the AboutActivity, but you have not provided the `android:parentActivityName` attribute here. Please consider adding it. You can look here for more details: [Providing Up navigation](https://developer.android.com/training/implementing-navigation/ancestral)